.github/workflows/mobile-e2e.yml

```
@@ -0,0 +1,177 @@
+    permissions:
+      contents: read
+    if: ${{ github.event.inputs.platform == 'android' || github.event.inputs.platform == 'both' || github.event_name == 'schedule' }}
+    
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v4
+
+      - name: 🏗 Setup pnpm
+        uses: pnpm/action-setup@v4
+        with:
+<<<<<<< HEAD
```

actionlint (1.7.10)

could not parse as YAML: could not find expected ':'

(syntax-check)